### ISSUE TITLE
Create a welcome email to new members

### ISSUE THREAD

**Issue by joesepi** [openjs-foundation/cross-project-council#464]
I think it would be good to have a welcome email for new members that alerts them to the actions taken when becoming a new member to the CPC (github org/team, email lists) and also reiterates expectations active participation as well as areas where privacy needs to be respected (i.e. new applications during silent period).

**Comment by brianwarner** [comment]
Yes, agreed.  New members of the mailing list get a welcome email.  Right
now it's fairly generic, but we can easily fix that if there's specific
text you'd like me to add.

**Comment by ijelliti** [comment]
@brianwarner that's good first issue to work on for me.

**Comment by HAsNightmare** [comment]
Hi everybody. I am Mrs. Jennifer Lynn Sherman, the infamous Nightmare that Dean Sherman is usually in a rotten a#@ mood about lol! Hopefully I learn enough to keep this annoying firewall troll from hacking me ever again and taking credit for all my developing. It has been going on for 6 long aggravating years and hopefully I can learn to protect myself in the near future.

**Comment by ovflowd** [comment]
Does this mailing list still exists? Haven't received any welcome e-mail, or with "members" we infer "project members"?

**Comment by ahadley1124** [comment]
@joesepi I will work on this welcome email.

**Comment by ahadley1124** [comment]
Here is a [link] (https://docs.google.com/document/d/1OVDUkI47xeARfdT50Ix-rLa052cfYseKY1Nwjg80Xg4/edit?usp=sharing) to the CPC welcome email. Please comment anything you would like to see on the email.

**Comment by joesepi** [comment]
As @brianwarner said above, there is an email that goes out when a new user is subscribed to an email, but there is only the most basic text in that email.

This is the platform we use to manage emails: https://lists.openjsf.org/g/main

I believe @lukeschantz may have been working on some good welcome text.

Here are some things that I think should be included in the welcome email:
- A link to Member Expectations in the CPC repo
- While we try to do as much as we can in public, there are instances where private matters are discussed in email/meetings/repos and confidentiality is important in these cases.
- An example of confidential matters: new project submissions to the foundation begin in a "silent period" and should not be discussed publicly. See here: https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#process
- A link to openjsf.org/collaborate which has a number of ways to be involved (Slack, calendar, github, etc)
- Join Slack if you haven't already and join the #cross-project-council channel.

I'll try to think of other things that would also be helpful...

**Comment by ovflowd** [comment]
Thank you for the insights, @joesepi!

**Comment by lukeschantz** [comment]
@ahadley1124 Please see draft of email that contains all the points mentioned in this thread: [](https://docs.google.com/document/d/1fa0oVIQZxkgMH6c_mOLMo1hX54KUlkqM1pSwxlN2Gsc/edit)

**Comment by sahilcode17** [comment]
is this issue still open ?

**Comment by bjohansebas** [comment]
Is that something we can do? Could the tool that manages emails in LF handle that? If it’s possible, then I can finish the welcome message and present it in a meeting to get approval, and have someone with sufficient permissions make it work.

If it’s not possible due to technical limitations, then it would be best to close this issue.

**Comment by bensternthal** [comment]
Closing this out. If there is "welcomish" information it should be easily found in the repo. If this is specific to onboarding voting members etc, we can document that in the repo.
